ariel/generate_reports.py

```diff
@@ -500,7 +500,7 @@ def get_units(instancetype):
 
     # GroupBy to assign appropriate index columns
     unused_az_ris = unused_az_ris.groupby(az_instance_groups.keys).sum()
-    ri_hourly_usage_report = ri_hourly_usage_report.groupby(region_instance_groups.keys + ['hourofweek']).sum()
+    ri_hourly_usage_report = ri_hourly_usage_report.groupby(region_instance_groups.keys + ['hourofweek']).sum(numeric_only=None)
```

Comment:
I am not incredibly well versed in Pandas but it looks to me from the docs that `numeric_only=None` is the default so this isn't needed (although this doesn't cause any issue either).

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sum.html

Reply:
Very close.  This is applying to a GroupBy, not a DataFrame, so the doc is https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.groupby.GroupBy.sum.html, which defaults to True.

I'm not sure why I needed to add this change.  When I looked at the source data I did not find any non-numeric data in the input.  My only guess was that it was reading some value like 0.13833299999999998 as non-numeric, which shouldn't be the case, but adding this flag fixed one of the many bugs I ran into earlier this week.

Reply:
Ok, cool - thanks for the clarification.